common/djangoapps/util/milestones_helpers.py

```diff
@@ -413,3 +414,16 @@ def add_user_milestone(user, milestone):
         return None
     from milestones import api as milestones_api
     return milestones_api.add_user_milestone(user, milestone)
+
+
+def get_milestones_service():
+    """
+    Returns MilestonesService instance if feature flag enabled;
+    else returns None.
+
+    Note: MilestonesService only has access to the functions
+    explicitly requested in the MilestonesServices class
+    """
+    if not settings.FEATURES.get('MILESTONES_APP', False):
+        return None
+    return MilestonesService(sys.modules[__name__])
```

Comment:
Why is passing `sys.modules[__name__]` needed?  The milestones code should be able to create the service without any parameter being passed.